### PR TITLE
Defined MySQL as the default rdbms for wallabag

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | yes ⚠️
| Tests pass?   | yes
| Documentation | no
| Translation   | no
| Fixed tickets | #2766
| License       | MIT

In fact, maybe we only need to changes parameters.yml.dist. @wallabag/core I want your opinion. 
We also need to update http://github.com/wallabag/doc/ I will do that after when this PR will be merged.

### CODE REVIEW

File: app/config/parameters.yml.dist [hunk 1]
```diff
@@ -10,17 +10,18 @@ parameters:
...
     database_user: root
     database_password: ~
-    database_path: "%kernel.root_dir%/../data/db/wallabag.sqlite"
+    # For SQLite, database_path should be "%kernel.root_dir%/../data/db/wallabag.sqlite"
+    database_path: null
     database_table_prefix: wallabag_
     database_socket: null
-    # with MySQL, use "utf8mb4" if you got problem with content with emojis
-    database_charset: utf8
+    # with PostgreSQL and SQLite, you can set "utf8"
```
Comment (tcitworld) [inline app/config/parameters.yml.dist]: must

**Comment by tcitworld** [comment]
If you'd ask me, I would have set Postgres as default. :P

**Comment by nicosomb** [comment]
utf8mb4 was introduced in MySQL 5.5.3: 2010-03-24 https://dev.mysql.com/doc/relnotes/mysql/5.5/en/news-5-5-3.html We can consider that everyone has recent MySQL server, right?

File: app/config/parameters.yml.dist [hunk 2]
```diff
@@ -10,17 +10,18 @@ parameters:
     # database_user: %env.database_user%
     # database_password: %env.database_password%
 
-    database_driver: pdo_sqlite
+    database_driver: pdo_mysql
     database_host: 127.0.0.1
     database_port: ~
     database_name: symfony
```
Comment (j0k3r) [inline app/config/parameters.yml.dist]: Can we also rename the default database to `wallabag`?

**Comment by tcitworld** [comment]
We could also change the `root` user to `wallabag`.

**Comment by nicosomb** [comment]
I already updated documentation repository (https://github.com/wallabag/doc/commit/c6610a3c163c7fb3398fe1ba9e1c2c9f7998fd33)

**Pull request merged**